packages/opensrp-plans/src/ducks/index.ts

```diff
@@ -0,0 +1,340 @@
+import { Dictionary } from '@onaio/utils';
+import intersect from 'fast_array_intersect';
+import { get, keyBy, values } from 'lodash';
+import { AnyAction, Store } from 'redux';
+import { createSelector } from 'reselect';
+import SeamlessImmutable from 'seamless-immutable';
+import { PlanDefinition } from '../plan-global-types';
+import { descendingOrderSort, isPlanDefinitionOfType } from '../helpers/utils';
+
+/** the reducer name */
+export const reducerName = 'PlanDefinition';
+
+/** list of plan types displayed
+ * TODO: To be moved to env
+ */
+export const DISPLAYED_PLAN_TYPES = 'FI,IRS,MDA,MDA-Point,Dynamic-FI,Dynamic-IRS,Dynamic-MDA';
```

Comment:
can this be passed to where its being used as a parameter/variable.

Reply:
@p-netm  Sure, I think we can best handle this once with clarity once we have the components that will receive this from the clients env

Reply:
ok, please create an issue to  revisit this @kahummer